src/sidebar/store/create-store.js

```diff
@@ -87,5 +101,5 @@ export default function createStore(modules, initArgs = [], middleware = []) {
 
   Object.assign(store, boundActions, boundSelectors);
 
-  return store;
+  return /** @type {SidebarStore} */ (store);
```

Comment:
Re-reading this PR I think it would make sense to move the `SidebarStore` type and cast to the `store` function, since `SidebarStore` includes all the specific modules in the sidebar's store whereas `createStore` is a utility where the type depends on which modules are passed as input.